### DISCUSSION TITLE
Update-2022-12-05

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#199] [Announcements]
Just for fun, implemented Conway's Game of Life.

https://user-images.githubusercontent.com/103860793/205758734-7c36a477-b221-4b04-b5fa-1de080fdea73.mov

**Comment by ghost** [comment]


**Reply by kaveh808** [reply]
Very cool. I will leave that as an exercise for the diligent reader. :)

**Reply by kaveh808** [reply]
I will check these out, thanks. Our code has had very little optimization (the graphics especially), so even a 100x100 grid has noticeable slowdown.

If you are interested in this project, please feel free to introduce yourself here: #8

**Reply by kaveh808** [reply]
Life in 3D.


https://user-images.githubusercontent.com/103860793/206061555-ddab5738-71b6-49ac-92b8-0a1a5d21aa30.mov

**Reply by aykaramba** [reply]
Amazing on so many levels.  Inspiring.

**Comment by kaveh808** [comment]
Added key bindings to create gliders.


https://user-images.githubusercontent.com/103860793/206318835-933f8e4c-1532-4ff2-9c09-37c2da3d7bb7.mov

**Comment by kaveh808** [comment]
3D repeaters.


https://user-images.githubusercontent.com/103860793/206330395-39a2f80c-ae40-4805-a5e4-244a951f8871.mov
